src/types/Policy/Policy.ts

```diff
@@ -1,34 +1,21 @@
 import { Action } from './Actions';
+import * as Generated from '../GeneratedOpenApi';
 
 export interface Policy {
-    id: string;
+    id: Generated.Uuid;
     actions: Action[];
     conditions: string;
-    customerid: string;
```

Comment:
How does it work if the server supplies it anyway?

Reply:
Actually it is marked as `@JsonbTransient` already and thus not returned. So we are good.